src/process-mining-visualisation/MiningView.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  svg,
+  RenderingContext,
+  Point,
+  EdgePadding,
+  IViewArgs,
+  IntersectingRoutedPoint,
+  PolylineEdgeViewWithGapsOnIntersections,
+  SEdgeImpl,
+  angleOfPoint,
+  toDegrees
+} from '@eclipse-glsp/client';
+import { VNode } from 'snabbdom';
+import { injectable } from 'inversify';
+import { Edge, EdgeLabel } from '@axonivy/process-editor';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const JSX = { createElement: svg };
+
+@injectable()
+export class MiningView extends PolylineEdgeViewWithGapsOnIntersections {
+  protected renderLine(edge: Edge, segments: Point[], context: RenderingContext): VNode {
+    if (edge.args && edge.args['labelvalue'] !== undefined && edge.children.length > 0) {
+      const label = edge.children[0];
```

Comment:
Why not use the builtin api for get the label of the edge? At least I think this should also work.
```tsx
const label = edge.editableLabel;
if (label) {
  label.text = "hi";
  ...
}

if the child has not the editlabel feature then you could at least do something similar as this code do it
```tsx
edge.children.find...
```
```